mimic/model/behaviors.py

```diff
@@ -85,7 +86,8 @@ class EventDescription(object):
     def declare_behavior_creator(self, name):
         """
         Decorator which declares that the decorated function is a factory,
-        taking parameters (a JSON-serialized object), and returning a behavior.
```

Comment:
This is weird… it was already only 79 characters, did you re-wrap it intentionally?